code/src/reducers/ui.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit'
```

Comment:
Super smurt to do separate reducers for the ui. Even though relatively simple logic in this app, it is good practice. We thunk it very nice. 👍 🧠 🤯